### PR TITLE
feat: various alterations to sunglasses so that NPC's won't have their heads melded with their sunglasses

### LINKED ISSUE #192: Sunglasses are melded into certain NPC's heads
Basically a continuation of a conversation between me and @Ocean-Minnow in #146 relating to sunglasses being misaligned.

### PR DESCRIPTION
**Pull request by dribbleondo**
fixes an issue raised in #146, fixes #192

the manifest has also been mostly alphabetised.

### CODE REVIEW

**Comment by dribbleondo** [comment]
Note: Some of the patches for the sunglasses contain "possible_duplicate" variants. I couldn't for the life of me work out which were the real ones for each NPC versus "fake" NPC's when looking in the games' files.

File: content/HeadMeldedSunglassesFix/chunk16/skunk_guard_VIPBodyguardJanus_M_HeroA_v4.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk16/skunk_guard_VIPBodyguardJanus_M_HeroA_v4.entity.patch.json:1]: This is the "Actor" variant - used by Janus' real guards

File: content/HeadMeldedSunglassesFix/chunk16/skunk_guard_VIPBodyguardJanus_M_HeroA_v4_possibly_duplicate.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk16/skunk_guard_VIPBodyguardJanus_M_HeroA_v4_possibly_duplicate.entity.patch.json:1]: This is the HeroA variant, used by 47 in disguise.

File: content/HeadMeldedSunglassesFix/chunk17/Mongoose_Unique_Kashmirian_HeroA_V0_0051B9B5211B6271.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk17/Mongoose_Unique_Kashmirian_HeroA_V0_0051B9B5211B6271.entity.patch.json:1]: This is an entitytemplate stored in chunk1. I'm not sure of its exact difference from the other one stored here but would need to be moved in order to do anything.

File: content/HeadMeldedSunglassesFix/chunk17/Mongoose_Unique_Kashmirian_HeroA_V0_008467369EE6DA15.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk17/Mongoose_Unique_Kashmirian_HeroA_V0_008467369EE6DA15.entity.patch.json:1]: This is an entitytype stored in chunk17. I'm not sure of its exact difference from the other one.

File: content/HeadMeldedSunglassesFix/chunk19/Flamingo_Guard_EventSecurityOfficer_M_HeroA_v2.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk19/Flamingo_Guard_EventSecurityOfficer_M_HeroA_v2.entity.patch.json:1]: Sunglasses will be placed on forehead when DIF is finished (and this actor is unused in vanilla anyway)

File: content/HeadMeldedSunglassesFix/chunk19/Flamingo_Guard_EventSecurityOfficer_M_HeroA_v4.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk19/Flamingo_Guard_EventSecurityOfficer_M_HeroA_v4.entity.patch.json:1]: Sunglasses will be placed on forehead when DIF is finished (and this actor is unused in vanilla anyway)

File: content/HeadMeldedSunglassesFix/chunk7/wolverine_unique_providencecommandoleader_m_heroa_v0.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk7/wolverine_unique_providencecommandoleader_m_heroa_v0.entity.patch.json:1]: entitytemplate in chunk2

File: content/HeadMeldedSunglassesFix/chunk7/wolverine_unique_providencecommandoleader_m_heroa_v0_possibly_duplicate.entity.patch.json
Comment (Ocean-Minnow) [inline content/HeadMeldedSunglassesFix/chunk7/wolverine_unique_providencecommandoleader_m_heroa_v0_possibly_duplicate.entity.patch.json:1]: entitytype in chunk7

**Pull request closed without merging**